src/avfoundation.cs

```diff
@@ -12488,8 +12519,18 @@ interface AVContentKeyRequest {
 		[Export ("processContentKeyResponseError:")]
 		void Process (NSError error);
 
+#if XAMCORE_4_0
+		[NoWatch, NoTV, NoMac]
+#elif !IOS
+		[Obsolete ("Not available and will be removed in a future version.")]
+#endif
+		[Deprecated (PlatformName.iOS, 11, 2, message: "Use the 'NSError' overload instead.")]
```

Comment:
it should not be both `Deprecated` (Apple change) and `Obsolete` (our changes)

Reply:
Change in headers, they added availability info, it was missing and also Deprecated iOS one in favour of the NSError one, I think this is the way to go because for iOS it is Deprecated and can be used if you are using iOS 11.0, OTOH, if you are using this selector lets say on AppleTV it was really never meant not to be used there hence the Obsolete.

Reply:
Same (empty stub) will also solve having a `[Deprecated]` on something that does not exists.